free-software-testing-books.md

```diff
@@ -5,7 +5,7 @@
 * [Testing of Free and OpenSource Software](#testing-of-free-and-opensource-software)
 * [Testing with different programming languages](#testing-with-different-programming-languages)
 * [Testing Tools](#testing-tools)
-* [Software verification](#software-verification)
+* [Formal Methods](#formal-methods)
```

Comment:
Changed because "Formal Methods" is a broader category than "Formal Verification", since FM also includes formal proofs, type systems, and specification.